src/components/name/more-detail.js

```diff
@@ -168,6 +168,26 @@ const AddMoreDeatil = (props) => {
                 </di>
               </div>
             </div>
+            <div className="col-lg-3">
+              <div className="features-icons-item mx-auto mb-0 mb-lg-3">
+                <div className="features-icons-icon d-flex">
+                  <i className="icon-list m-auto text-primary"></i>
+                </div>
+                <h3>What is the name that you would like to enter?</h3>
+                <input
+                  onChange={(e) => setName(e.target.value)}
+                  className="form-control"
+                />
+                <di>
```

Comment:
I would advise working while the console is open. It will tell you what you have you missed. You want to have the div to be properly handled.